public/js/modMap.js

```diff
@@ -0,0 +1,172 @@
+///Modmap.js
+
+let article = document.querySelector('#modMap');
+console.log(article.dataset);
+let data = JSON.parse(article.dataset.mods)
+
+function checkIfModExists(data, x, y) {
+  var i, len = data.length;
+
+  for (i = 0; i < len; i++) {
+    if (data[i]) {
+      console.log('data',data[i]);
+      if (data[i]["y"] == y && data[i]["x"] == x) {
+        return data[i]
+      }
+    }
+  }
+
+  return false;
+}
+
+function modMap() {
+  var draw = SVG().addTo('#modMap').size(400, 3800)
+  var article = document.querySelector('#modMap');
+  console.log(article.dataset);
+  var allMods = JSON.parse(article.dataset.mods);
+  var defaultColor = 'grey'
+  console.log(allMods[0]["x"]);
```

Comment:
there's a chance that this `console.log` statement could break the whole script because `allMods[0]` could be null or undefined - I would recommend either printing out all of `allMods` if you're really interested in debugging it or checking to make sure `allMods[0]` exists first